### PR TITLE
Changelog for Docker Desktop 2.5.4

### PR DESCRIPTION
**Pull request by lorenrh**
Signed-off-by: Lorena Rangel <lorena.rangel@docker.com>

### Proposed changes

Changelog for Docker Desktop 2.5.4 release

### CODE REVIEW

**Review by StefanScherer — DISMISSED** [review]
LGTM

File: docker-for-mac/edge-release-notes.md [hunk 1]
```diff
@@ -11,6 +11,20 @@ This page contains information about Docker Desktop Edge releases. Edge releases
...
+> [Download](https://desktop.docker.com/mac/edge/50534/Docker.dmg)
+
+### Upgrades
+
+- [Docker Engine 20.10.0-rc2](https://github.com/docker/docker-ce/blob/master/CHANGELOG.md#20100)
+
+### Bug fixes and minor changes
+
+- Changed the «Update and quit» menu entry to «Update and restart».
+- Fixed the manual check for updates dialog reporting a new available version number being the build number instead of the version number.
```
Comment (stephen-turner) [inline docker-for-mac/edge-release-notes.md:26]: Too wordy. Try:

Fixed the check for updates dialog reporting the build number instead of the version number of a new version.

**Review by stephen-turner — CHANGES_REQUESTED** [review]
One change requested inline, but we also need mention of the CPU fix.

File: docker-for-mac/edge-release-notes.md (hunk 1 above)
Comment (lorenrh) [reply docker-for-mac/edge-release-notes.md]: Done! 😄

**Review by mat007 — DISMISSED** [review]
Looks fine!

**Review by StefanScherer — APPROVED** [review]

File: docker-for-mac/edge-release-notes.md [hunk 2]
```diff
@@ -11,6 +11,24 @@ This page contains information about Docker Desktop Edge releases. Edge releases
...
+### Upgrades
+
+- [Docker Engine 20.10.0-rc2](https://github.com/docker/docker-ce/blob/master/CHANGELOG.md#20100)
+- [Go 1.15.6](https://github.com/golang/go/issues?q=milestone%3AGo1.15.6+label%3ACherryPickApproved+)
+
+### Bug fixes and minor changes
+
+- Changed the «Update and quit» menu entry to «Update and restart».
+- Fixed the check for updates dialog reporting the build number instead of the version number of a new version.
+- Downgrade the kernel to [4.19.121](https://hub.docker.com/layers/docker/for-desktop-kernel/4.19.121-2a1dbedf3f998dac347c499808d7c7e029fbc4d3-amd64/images/sha256-4e7d94522be4f25f1fbb626d5a0142cbb6e785f37e437f6fd4285e64a199883a?context=repo) to reduce the CPU usage of hyperkit. Fixes [docker/for-mac#5044](https://github.com/docker/for-mac/issues/5044)
```
Comment (stephen-turner) [inline docker-for-mac/edge-release-notes.md:28]: Downgraded

File: docker-for-mac/edge-release-notes.md [hunk 3]
```diff
@@ -11,6 +11,24 @@ This page contains information about Docker Desktop Edge releases. Edge releases
...
+-  DNS: avoid returning `NXDOMAIN` when a name exists but the type of records was not found. Fixes [docker/for-mac#5020](https://github.com/docker/for-mac/issues/5020). Related to https://gitlab.alpinelinux.org/alpine/aports/-/issues/11879
```
Comment (stephen-turner) [inline docker-for-mac/edge-release-notes.md:29]: Fixed a bug that DNS would return `NXDOMAIN` when a name exists but the type of record was not found.

File: docker-for-windows/edge-release-notes.md [hunk 4]
```diff
@@ -11,6 +11,21 @@ This page contains information about Docker Desktop Edge releases. Edge releases
...
+
+### Upgrades
+
+- [Docker Engine 20.10.0-rc2](https://github.com/docker/docker-ce/blob/master/CHANGELOG.md#20100)
+- [Go 1.15.6](https://github.com/golang/go/issues?q=milestone%3AGo1.15.6+label%3ACherryPickApproved+)
+
+### Bug fixes and minor changes
+
+- Fixed an issue causing the background update to fail when the current user is not an administrator.
+- Downgrade the kernel to [4.19.121](https://hub.docker.com/layers/docker/for-desktop-kernel/4.19.121-2a1dbedf3f998dac347c499808d7c7e029fbc4d3-amd64/images/sha256-4e7d94522be4f25f1fbb626d5a0142cbb6e785f37e437f6fd4285e64a199883a?context=repo) to reduce the CPU usage of hyperkit. Fixes [docker/for-mac#5044](https://github.com/docker/for-mac/issues/5044)
```
Comment (stephen-turner) [inline docker-for-windows/edge-release-notes.md:27]: Downgraded

**Review by stephen-turner — CHANGES_REQUESTED** [review]
Latest version looks good, just a couple of nits

**Review by djs55 — DISMISSED** [review]

File: docker-for-windows/edge-release-notes.md (hunk 4 above)
Comment (lorenrh) [reply docker-for-windows/edge-release-notes.md]: done

File: docker-for-mac/edge-release-notes.md [hunk 5]
```diff
@@ -11,6 +11,24 @@ This page contains information about Docker Desktop Edge releases. Edge releases
...
+
+- [Docker Engine 20.10.0-rc2](https://github.com/docker/docker-ce/blob/master/CHANGELOG.md#20100)
+- [Go 1.15.6](https://github.com/golang/go/issues?q=milestone%3AGo1.15.6+label%3ACherryPickApproved+)
+
+### Bug fixes and minor changes
+
+- Changed the «Update and quit» menu entry to «Update and restart».
+- Fixed the check for updates dialog reporting the build number instead of the version number of a new version.
+- Downgraded the kernel to [4.19.121](https://hub.docker.com/layers/docker/for-desktop-kernel/4.19.121-2a1dbedf3f998dac347c499808d7c7e029fbc4d3-amd64/images/sha256-4e7d94522be4f25f1fbb626d5a0142cbb6e785f37e437f6fd4285e64a199883a?context=repo) to reduce the CPU usage of hyperkit. Fixes [docker/for-mac#5044](https://github.com/docker/for-mac/issues/5044)
+-  Fixed a bug that DNS would return `NXDOMAIN` when a name exists but the type of record was not found.. Fixes [docker/for-mac#5020](https://github.com/docker/for-mac/issues/5020). Related to https://gitlab.alpinelinux.org/alpine/aports/-/issues/11879
```
Comment (stephen-turner) [inline docker-for-mac/edge-release-notes.md:29]: Teeny tiny nit: you have two full stops at the end of the first sentence.

**Review by stephen-turner — CHANGES_REQUESTED** [review]

File: docker-for-mac/edge-release-notes.md (hunk 5 above)
Comment (lorenrh) [reply docker-for-mac/edge-release-notes.md]: Just to make sure the sentence stopped. It's fixed now! 😄

**Review by stephen-turner — APPROVED** [review]
Thanks, approving now.
